### DISCUSSION TITLE
Stats on downloads from https://cataclysmdda.org/ for releases vs experimental?

### DISCUSSION THREAD

**Discussion by actual-nh** [CleverRaven/Cataclysm-DDA#48107] [Q&A] [answered]
Hi. I'm doing some work on the [CDDA main wiki](http://cddawiki.chezzo.com/cdda_wiki/index.php), and it would be useful to have some idea of whether players are more downloading the most recent release, or an experimental version. Any other information available about proportions playing each would also be nice. Why? It helps us know what version the wiki should be focused on. Ping: @kevingranade? Thanks!

**Comment by Zireael07** [comment]
Pinging @narc0tiq since he maintains the buildbot IIRC - maybe he has some way to track download stats?

**Reply by actual-nh** [reply]
Thanks!

**Comment by narc0tiq** [comment]
Releases are hosted by Github -- you can find all of them at https://github.com/CleverRaven/Cataclysm-DDA/releases -- and I recall the API does expose download counts for each artifact of each release. I don't think the API shows any kind of history, but there might be analytics for that hiding somewhere in the Github UI (but probably not).

You'll also have to figure out how to aggregate the data from the many experimental releases versus the relatively few full releases -- someone who plays experimentals probably doesn't download all of them, whereas it's relatively normal for someone who plays stable to have all the stable versions. Likewise, there's only counts of downloads, with no attempt to aggregate the data for the same person downloading the same thing multiple times.

**Reply by actual-nh** [reply]
Thanks! I'll take a look at that - probably this weekend. I'm mostly interested if it shows a clear preponderance one way or the other. Tracking the history from now till sometime after 0.F may be needed if there's not a clear preponderance.

**Comment by ZhilkinSerg** [comment, marked as answer]
Use http://mmilidoni.github.io/github-downloads-count/

Filename | Downloads 
-- | -- 
cataclysmdda-0.E-Windows_x64-Tiles-b11511.zip | 1336 
cataclysmdda-0.E-Windows_x64-Tiles-b11514.zip | 1318 
cataclysmdda-0.E-Windows_x64-Tiles-b11519.zip | 1178 
cataclysmdda-0.E-Windows_x64-Tiles-b11521.zip | 828 
cataclysmdda-0.E-Windows_x64-Tiles-b11529.zip | 821 
cataclysmdda-0.E-Windows_x64-Tiles-b11524.zip | 802 
cataclysmdda-0.E-Windows_x64-Tiles-b11527.zip | 721 
cataclysmdda-0.E-Windows_x64-Tiles-b11531.zip | 582 
cataclysmdda-0.E-Windows_x64-Tiles-b11518.zip | 470 
cataclysmdda-0.E-Windows_x64-Tiles-b11523.zip | 410 
cataclysmdda-0.E-Windows_x64-Tiles-b11508.zip | 408 
cataclysmdda-0.E-Windows_x64-Tiles-b11507.zip | 262 
cataclysmdda-0.E-Windows_x64-Tiles-b11533.zip | 251 
cataclysmdda-0.E-Windows_x64-Tiles-b11516.zip | 248 
cataclysmdda-0.E-Windows_x64-Tiles-b11513.zip | 221 
cataclysmdda-0.E-Windows_x64-Tiles-b11510.zip | 143 
cataclysmdda-0.E-Windows_x64-Tiles-b11526.zip | 142 
cataclysmdda-0.E-Windows-Tiles-b11514.zip | 79 
cataclysmdda--Android_arm64-Tiles-b11521.apk | 73 
cataclysmdda--Android_arm64-Tiles-b11514.apk | 73 
cataclysmdda-0.E-Windows-Tiles-b11511.zip | 69 
cataclysmdda--Android_arm64-Tiles-b11511.apk | 54 
cataclysmdda-0.E-Linux_x64-Tiles-b11514.tar.gz | 53 
cataclysmdda-0.E-Windows-Tiles-b11521.zip | 48 
cataclysmdda-0.E-Windows-Tiles-b11524.zip | 42 
cataclysmdda-0.E-Linux_x64-Tiles-b11527.tar.gz | 39 
cataclysmdda-0.E-Windows-Tiles-b11527.zip | 39 
cataclysmdda--Android_arm32-Tiles-b11514.apk | 38 
cataclysmdda-0.E-Windows-Tiles-b11520.zip | 32 
cataclysmdda--Android_arm32-Tiles-b11521.apk | 29 
cataclysmdda-0.E-Linux_x64-Tiles-b11511.tar.gz | 28 
cataclysmdda-0.E-Linux_x64-Curses-b11527.tar.gz | 26 
cataclysmdda-0.E-Linux_x64-Tiles-b11521.tar.gz | 26 
cataclysmdda--Android_arm32-Tiles-b11511.apk | 26 
cataclysmdda-0.E-Windows-Tiles-b11510.zip | 23 
cataclysmdda-0.E-OSX-Tiles-b11514.dmg | 22 
cataclysmdda-0.E-Windows-Tiles-b11507.zip | 22 
cataclysmdda-0.E-Windows-Tiles-b11530.zip | 21 
cataclysmdda-0.E-Windows-Tiles-b11508.zip | 21
